docs/AssignmentEntity.html

```diff
@@ -2568,29 +2568,28 @@ <h5>Returns:</h5>
         Type
     </dt>
     <dd>
-        
+
 <span class="param-type">string</span>
 
 
     </dd>
 </dl>
 
-    
 
 
 
 
 
-        
-            
 
-    
 
-    
-    <h4 class="name" id="instructionsRichTextEditorConfig"><span class="type-signature"></span>instructionsRichTextEditorConfig<span class="signature">()</span><span class="type-signature"> &rarr; {object}</span></h4>
```

Comment:
rip all these huge diffs...most are spacing changes only, seems like this is the only change in this file?